.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,88 @@
+name: Release docs
```

Comment:
An idea to maybe simplify this and reduce some duplication:

What if this job only updates the existing release assets (specifically the docs tarball), and then triggers the docsite update directly like https://github.com/winglang/wing/blob/main/.github/workflows/update-docsite.yml

If the only change is pure docs, I think it's safe to avoid versioning/releasing as long as the documentation site is told to update.

Reply:
This is already downloading existing assets and modifying only the tarball.
In regards to the last part, I believe that was my initial suggestion; skipping release. But at this point I'd lean towards not skipping the release because while the workflow is not entirely new, it gives a fast and proper solution that'd not affect the existing release logic. What do you say?

Reply:
Ah my bad, I misread some of the changes. Nice!

The original reason to avoid skipping a release was just to ensure the docsite gets updated. I'm cool with this approach.